src/content/tutorials/connectivity/ingress/multi-nginx-ic/index.md

```diff
@@ -0,0 +1,151 @@
+---
+linkTitle: Multiple ingress nginx controllers
+title: Running multiple ingress nginx controllers
+description: Deploy multiple ingress nginx controllers in a Kubernetes cluster to separate different ingress traffic classes.
+weight: 20
+menu:
+  principal:
+    parent: tutorials-connectivity-ingress
+    identifier: tutorials-connectivity-ingress-multi-ic
+user_questions:
+  - How do I install multiple ingress nginx controllers?
+  - How do I separate internal and external Services?
+  - How do I configure ingress nginx controller for internal traffic?
+  - How do I override the NodePorts on KVM ingresses?
+  - How do I configure ingress nginx controller to allow weak ciphers?
+last_review_date: 2024-08-26
+aliases:
+  - /advanced/connectivity/ingress/multi-nginx-ic
+  - /guides/multi-nginx/
+  - /advanced/multi-nginx/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-cabbage
+---
+
+ingress nginx controller handles [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources, routing traffic from outside the Kubernetes cluster to services within the cluster.
+
+It is possible to install multiple ingress controllers in a Kubernetes cluster. The ingress nginx controller can be [installed as an App on your cluster]({{< relref "/getting-started/install-an-application#install-ingress-controller" >}}).
+ingress nginx installs an `IngressClass` with default name `nginx` and controller value `k8s.io/ingress-nginx`.
+
+Some use cases for this might be:
+
+- An ingress controller that's behind an internal ELB for traffic between services within the VPC (or a group of peered VPCs)
+- An ingress controller behind an ELB that already terminates SSL
+- An ingress controller with different functionality or performance
+
```

Comment:
```suggestion
It's possible to install multiple ingress controllers in a Kubernetes cluster. The ingress nginx controller can be [installed as an App on your cluster]({{< relref "/getting-started/install-an-application#install-ingress-controller" >}}). Ingress nginx installs an `IngressClass` with the default name `nginx` and controller value `k8s.io/ingress-nginx`.

Some use cases for this might be:

- An ingress controller that's behind an internal ELB for traffic between services within the VPC (or a group of peered VPCs)
- An ingress controller behind an ELB that already terminates SSL
- An ingress controller with different functionality or performance

```